javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -925,7 +953,11 @@ class SinkPathNode extends PathNode {
  */
 module PathGraph {
   /** Holds if `nd` is a node in the graph of data flow path explanations. */
-  query predicate nodes(PathNode nd) { any() }
+  query predicate nodes(PathNode nd) {
+    not nd.isHidden() or
+    nd instanceof SourcePathNode or
```

Comment:
Edge case: the `SourcePathNode` character is independent of the path. So you may display a hidden `SourcePathNode` in the middle of your path. I suppose we can live with that.

Reply:
I'm fine with that.